### PR TITLE
🌱 Uplift CAPI to v1.3.0

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

File: docs/releasing.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ You must have docker installed.
...
 1. A git tag
 1. A release on Github containing:
-    - A manifest file - `infrastructure-components.yaml`
-    - A metadata file - `metadata.yaml`
-    - A cluster template - `cluster-template.yaml`
-    - A file containing an example of variables to set - `example_variables.rc`
+  - A manifest file - `infrastructure-components.yaml`
+  - A metadata file - `metadata.yaml`
+  - A cluster template - `cluster-template.yaml`
+  - A file containing an example of variables to set - `example_variables.rc`
```
Comment (lentzi90) [inline docs/releasing.md:32]: This does not look correct now as rendered by github. I think that extra space is needed.
image

**Review by lentzi90 — COMMENTED** [review]
Just one nit. Otherwise it looks good

File: docs/releasing.md (hunk 1 above)
Comment (kashifest) [reply docs/releasing.md:32]: True, let me check, thanks for noticing.

**Review by furkatgofurov7 — APPROVED** [review]
/approve
/hold 
to address the comments. Also, please make sure to run e2e tests on this one to make sure the uplift is not breaking the tests.

File: docs/releasing.md (hunk 1 above)
Comment (kashifest) [reply docs/releasing.md]: Done

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Pull request merged**